package.json

```diff
@@ -300,10 +301,11 @@
     "i18n-check-empty-src": "npm run mmjstool -- i18n check-empty-src --webapp-dir .",
     "storybook": "start-storybook -p 6006 -s ./storybook/static",
     "build-storybook": "build-storybook -s ./storybook/static",
-    "check-types": "tsc",
+    "check-types": "tsc -b",
```

Comment:
This is required for TypeScript to support the [project references](https://www.typescriptlang.org/docs/handbook/project-references.html) that we're using to have the client package reference the types package